ast/compile_test.go

```diff
@@ -2351,6 +2381,30 @@ func assertNotFailed(t *testing.T, c *Compiler) {
 	}
 }
 
+func checkUnsafeFunctionCall(c *Compiler) *Error {
```

Comment:
Let's remove this from this PR and just use a mock function above to exercise the stage hooks.

Reply:
Updated test.